examples/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">=1.1.6"
+  required_version = ">=1.3"
```

Comment:
@dhutty I've merged #15 to main branch. Could you please revert terraform upgrade changes from this PR.